core/src/dispatch/mod.rs

```diff
@@ -125,6 +129,18 @@ impl NetworkConfig {
     pub fn get_custom_allocator(&self) -> &Option<Arc<dyn ChunkAllocator>> {
         &self.custom_allocator
     }
+
+    /// Reads the tcp_nodelay parameter of the NetworkConfig
+    pub fn get_tcp_nodelay(&self) -> bool {
+        self.tcp_nodelay
+    }
+
+    /// If set to true the Nagle algorithm will be turned off for all TCP Network-channels
```

Comment:
Leave a newline after the first line.

Write `false` (with backticks) instead of FALSE